src/app/components/poptip/poptip.component.html

```diff
@@ -0,0 +1,11 @@
+<popper-content #poptip>
+    <p>{{msg}}</p>
+</popper-content>
+<div class="info-icon"  [popper]="poptip"
+                        [popperTrigger]="'hover'"
```

Comment:
Keep in mind that some settings can be set as system-wide defaults in `NgxPopperModule.forRoot({})`